src/components/DateTimePicker/Internal/OcPicker.types.ts

```diff
@@ -745,6 +787,20 @@ export type OcPickerSharedProps<DateType> = {
    * The picker shape.
    */
   shape?: DatePickerShape | Shape;
+  /**
+   * Show the 'Now' button in partial when `showTime` is set.
+   * @default true
+   */
+  showNow?: boolean;
+  /**
+   * Show the 'OK' button
+   * @default true
+   */
+  showOk?: boolean;
+  /**
+   * Show the `Today` button.
```

Comment:
oh. its a button. probably disregard previous comment. Would it be sensible to add the word "Button" to these show* props?

Reply:
Will update in Octuple v3.